### DISCUSSION TITLE
Is it possible to export/import projects between Gears and SPIKE?

### DISCUSSION THREAD

**Discussion by mbockus** [QuirkyCort/gears#92] [Q&A] [answered]
Is it possible to switch projects between the SPIKE app and the gears simulator? 

Thanks for building this simulator! It's going to be a huge help to get the team more programming practice without having to have access to a robot/table.

**Comment by QuirkyCort** [comment, marked as answer]
Unfortunately, no. The blocks are not the same between Gears and SPIKE.

It's shouldn't be too hard to do, as the SPIKE app appears to be using Blockly as well; just a matter of adding the blocks definition in Gears and writing the code generator. This isn't an area of interest for me, but maybe someone else will take it up.
